packages/server/src/routes/grants.js

```diff
@@ -29,6 +32,24 @@ async function getAgencyForUser(selectedAgency, user, { filterByMainAgency } = {
     return agencies.map((s) => s.id);
 }
 
+// Award floor field was requested for CSV export but is not stored as a dedicated column,
+// so we have to extract it from raw_body
+function getAwardFloor(grant) {
```

Comment:
If we wanted to add it, this is where https://github.com/usdigitalresponse/usdr-gost/blob/main/packages/server/src/lib/grantscraper/index.js#L29

Reply:
Cool. I prob won't bother with this for now since this is the only consumer and we don't have a need to query against it in the DB, but good to know. If I were to add a new column and fill it there, will existing rows get it backfilled from future grantscraper runs? Or will it only be filled on newly inserted rows?

Reply:
hmm not sure but I believe so since it will update or insert the grant